data-model/fdo-profiles/0.1.0/mas/schema/mas-request.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$id": "https://json-schema.org/draft/2020-12/schemas/fdo-profiles/0.1.0/mas-request.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "FDO Profile requests to PID API for Machine Annotation Services",
+  "allof": [
+    {
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/handle-request.json"
+    }
+  ],
+  "masName": {
+    "type": "string",
+    "description": "Name of Machine Annotation Service",
+    "$comment": "idx = 604"
+  },
+  "additionalProperties": false,
+  "required": [
+    "masName"
```

Comment:
is PID of the mas not required?

Reply:
Just above, we have: 
```
"allof": [
    {
      "$ref": "https://schemas.dissco.tech/schemas/fdo-profiles/0.1.0/handle-kernel.json"
    }
  ]
```
This means it must also adhere to the pid kernel metadata schema, which includes the PID name.